tests/results/result_actions/OpenEditorActionTest.py

```diff
@@ -152,6 +152,12 @@ def test_environ_editor(self):
             handle.writelines(file_dict[self.fb])
         result = Result.from_values('origin', 'msg', self.fb)
 
+        # Some users have the ``EDITOR`` environment variable set, so
+        # we should remove that temporarily.
```

Comment:
The "temporarily" doesn't really apply here, we never set it back. I think just "we should remove it, to test the behavior if it is missing" or something should suffice